Pagination.astro

```diff
@@ -10,6 +10,7 @@ interface Props {
   lastPage?: RouteString
   currentPage?: string | number
   totalPages?: NumericalString | number
+  renderProgress?: ({ currentPage, totalPages }: { currentPage: string; totalPages: string }) => string
```

Comment:
@boredland I'm getting a type error at the moment, since `currentPage` and `totalPages` can be a (numerical)string or number, we should also define that here as well:

```suggestion
  renderProgress?: ({ currentPage, totalPages }: { currentPage: string | number; totalPages: string | number }) => string
```